Introduction/edits for clarity (#591)
data-edu/data-science-in-education: 01-introduction.Rmd

**Abstract**

> This chapter is a welcome to readers and an overview of the book. It describes challenges with learning data science in education, including the complexity of the data scientist role. Finally, this chapter introduces data analysis walkthroughs as a way to learn data science in the education field.

Reviewer: Super minor - should we change "data analysis walkthroughs" to "walkthroughs"? Reasoning: data analysis != data science, and this is simpler

Dear Data Scientists, Educators, and Data Scientists who are Educators: 

> This book is a warm welcome and an invitation to learn with a great community. If you're a data scientist in education or an educator in data science, your role probably isn't straightforward. This book is our contribution to a growing movement to merge the paths of data analysis and education. We wrote this book to make your first step on that path clearer and less scary.

Reviewer 2: ```suggestion
This book is a warm welcome and an invitation to join a great community. If you're a data scientist in education or an educator in data science, your role probably isn't straightforward. This book is our contribution to a growing movement to merge the paths of data analysis and education. We wrote this book to provide clarity and support on your first steps in this direction.
```

> This book is a warm welcome and an invitation to learn with a great community. If you're a data scientist in education or an educator in data science, your role probably isn't straightforward. This book is our contribution to a growing movement to merge the paths of data analysis and education. We wrote this book to make your first step on that path clearer and less scary.

Reviewer: very minor, we should norm on using contractions (e.g., "isn't"). I dig it, but I think we are inconsistent throughout.

> This book is a warm welcome and an invitation to learn with a great community. If you're a data scientist in education or an educator in data science, your role probably isn't straightforward. This book is our contribution to a growing movement to merge the paths of data analysis and education. We wrote this book to make your first step on that path clearer and less scary.

Author: @jrosen48 good flag on the contractions. I'll put it in the agenda for the next meeting and also message in Slack.

> This book is a warm welcome and an invitation to learn with a great community. If you're a data scientist in education or an educator in data science, your role probably isn't straightforward. This book is our contribution to a growing movement to merge the paths of data analysis and education. We wrote this book to make your first step on that path clearer and less scary.

Author: @ivelasq good flag on the definition of community. My first thought is community means other people learning R in the education field. But you're right—it's hard to point to where that is. I'll play around with more general wording.

> Whether you're a data scientist working in education or an educator wanting to learn data science, we invite you to read this book and put these techniques to work in the real world. Your work in the education community will help decide how education and data science come together.

Reviewer 2: Maybe "offers practical techniques for real-world application"

> Whether you're a data scientist working in education or an educator wanting to learn data science, we invite you to read this book and put these techniques to work in the real world. Your work in the education community will help decide how education and data science come together.

Reviewer 2: Is education a community or a field?

## Learning data science in education

> Over the coming chapters, you'll explore examples of what data science in education looks like. But first, let's talk about why data science in education is not easily definable. 

Reviewer 2: ```suggestion
In the coming chapters, you'll explore examples of data science in education. But first, let's talk about why data science in education is not easily definable. 
```

> Being a data scientist in education is challenging because there is not a universal vision of what that means. For starters, education as a field is complex. If education were a building, it would be multi-storied with many rooms. There are privately and publicly funded schools. There are more than 18 possible grade levels. Students can learn alone or with others in a classroom. Students start school, leave school, finish school, and return to school. 

Reviewer 2: ```suggestion
Being a data scientist in education is challenging because there is no universal vision of what that means. Education is a complex field. If education were a building, it would be multi-storied with countless rooms. There are privately and publicly funded schools with more than 18 possible grade levels. Students learn alone or with others in a classroom. Their journeys involve starting, leaving, finishing, and even returning to school. 
```

> This imaginary building we call education also has rooms most never see -- rooms where business and finance staff plan the most efficient use of limited funds. The transportation department plans bus routes across vast spaces. University administrators search for the best way to measure career readiness. Education consultants study how students perform in coursework and how they feel about class materials. 

Reviewer 2: ```suggestion
This metaphorical building we call education also has rooms most never see -- rooms where business and finance staff plan the most efficient use of limited funds. The transportation department plans bus routes across vast spaces. University administrators search for the best way to measure career readiness. Education consultants study student performance and sentiments. 
```

There are many ways one *could* do data science in education, but consensus on ways one *should* do it is just starting. The community is still working out how it all fits together. 

> Even then, there's still the issue of how education systems can best use data science tools. In many education settings, school staff have never worked with someone with this combination of skills -- education content knowledge, programming, and statistics [@conway2010data].

Reviewer 2: ```suggestion
Even then, there's still the challenge of how education systems can best use data science tools. In many education settings, school staff have never worked with someone with the blend of skills encompassing education content knowledge, programming, and statistics --- the "Data Science Venn Diagram" [@conway2010data].
```

> ## Making the path clearer

Reviewer: I am wondering if somewhere here (or in the previous paragraph -- or earlier) we offer up a definition of educational data science. I like the definition of McFarland et al. (2021) -- an expansive definition that conceives of educational data science as "an umbrella for a range of new and often nontraditional quantitative methods (such as machine learning, network analysis, and natural language processing) applied to educational problems often using novel data" (p. 2). 

@article{mcfarland2021education,
  title={Education data science: Past, present, future},
  author={McFarland, Daniel A and Khanna, Saurabh and Domingue, Benjamin W and Pardos, Zachary A},
  journal={AERA Open},
  volume={7},
  pages={23328584211052055},
  year={2021},
  publisher={SAGE Publications Sage CA: Los Angeles, CA}
}

Author: I added this definition and reference. Thanks!

As data science in education grows, the way we think about it also needs to grow. Doing so can help us advance data science in education as a discipline. It will uncover the unique opportunities that come with analyzing data in our domain.

> With this chapter, we begin by offering a primer for data science in education, including a discussion of foundational skills in the programming language R. The primer continues with suggestions for how to use this text ([Chapter 2](#c02)), our definition of the data science process, what it looks like in different roles ([Chapter 3](#c03)), and a discussion of similarities and differences between data science in education and data science in others fields ([Chapter 4](#c04)).

Reviewer 2: I feel that the original was more accurate.

> With this chapter, we begin by offering a primer for data science in education, including a discussion of foundational skills in the programming language R. The primer continues with suggestions for how to use this text ([Chapter 2](#c02)), our definition of the data science process, what it looks like in different roles ([Chapter 3](#c03)), and a discussion of similarities and differences between data science in education and data science in others fields ([Chapter 4](#c04)).

Author: Makes sense. Changed it back. TY.

> Next, you'll take what you've learned and apply it in education data analysis walkthroughs. The walkthroughs are meant to be an example-driven approach to learning. They include recognizable and actionable demonstrations. 

Reviewer 2: ```suggestion
Next, you'll take what you've learned and apply it in data analysis walkthroughs using education data. The walkthroughs are meant to aid learning through practical examples. They include recognizable and actionable demonstrations. 
```

Reviewer: Should we call these "walkthroughs" or "data science walkthroughs"? Does data analysis = data science?

The chapters in this book fall into four different themes:

**Build a foundation to use R and RStudio**

* [Getting Started with R and RStudio](#c05)
* [Foundational Skills](#c06)

**Analyzing student perception data**

* [Walkthrough 1: The Education Dataset Science Pipeline](#c07)
* [Walkthrough 5: Text Analysis With Social Media Data](#c16)
* [Walkthrough 7: The Role (and Usefulness) of Multilevel Models](#c13)

**Analyzing student performance data**

* [Walkthrough 2: Approaching Gradebook Data From a Data Science Perspective](#c08)
* [Walkthrough 8: Predicting Students' Final Grades Using Machine Learning Methods](#c14)

**Using publicly available data**

* [Walkthrough 3: Introduction to Aggregate Data](#c09)
* [Walkthrough 4: Longitudinal Analysis With Federal Students With Disabilities Data](#c10)

> We'll end the book by discussing how to bring data science skills into your education job ([Chapter 15](#c15), an overview of teaching data science ([Chapter 16])(#c16)), and supplemental resources ([Chapter 17](#c17) and [Chapter 18](#c18)). 

Reviewer: will defer to you and to @restrellado here, but agree we should make this consistent

> We'll end the book by discussing how to bring data science skills into your education job ([Chapter 15](#c15), an overview of teaching data science ([Chapter 16])(#c16)), and supplemental resources ([Chapter 17](#c17) and [Chapter 18](#c18)). 

Author: Thank you. I made some changes so we use present tense more consistently. Only exceptions are where future and past tense made more sense. Also noted as an agenda item for future discussion.

> We hope after reading this book, you'll feel less alone in your journey to learn data science. We also hope your experience with this book is both challenging and fun. Finally, we hope you'll take what you learned and share it with others.

Reviewer 2: ```suggestion
We hope after reading this book, you'll feel less alone in your data science journey. We also hope your experience with this book is both challenging and fun. Finally, we hope you'll take what you learned and share it with others.
```
